packages/apps/dev-wallet/src/App/Layout/Layout.tsx

```diff
@@ -26,7 +27,7 @@ import {
 export const Layout: FC = () => {
   const { theme, setTheme } = useTheme();
   const location = useLocation();
-  const navigate = useNavigate();
+  const navigate = useTheCorrectNavigate();
```

Comment:
Why is this name like that 😂?

```suggestion
  const navigate = usePatchedNavigate();
```

Reply:
I was about to name it `useTheFuckingNavigation` but changed it at the last moment ;)